tests/test_operation_definition.py

```diff
@@ -199,3 +199,18 @@ def test_attribute_accessors():
 
     op = AttributeOp.create(attributes={"attr": StringAttr("test")})
     assert op.opt_attr is None
+
+
+def test_attribute_setters():
```

Comment:
Can be parameterised

Reply:
I personally think here that the test is way more readable this way, rather than creating two tests (one for optional and one for non-optional), and then parametrizing it just for the optional one (because there are only two cases, it's a bit useless to test it on more arbitrary strings).
What do you think?

Reply:
Oh I see now. It is fine as it is, yes. Sorry